infra/hasura/metadata/build-metadata.sh

```
@@ -0,0 +1,160 @@
+    
+
+    if [ -f "$TENANTS_DIR/$tenant/tenant_overrides.yaml" ]; then
+        echo "Applying tenant overrides..."
+      
+        find "$BUILD_DIR/$tenant" -name "*.yaml" -type f | while read -r yaml_file; do
+            relative_path="${yaml_file#$BUILD_DIR/$tenant/}"
+            yaml_path=$(dirname "$relative_path")
+            
+           
+            $YAML_MERGE_TOOL eval ".\"$yaml_path\"" "$TENANTS_DIR/$tenant/tenant_overrides.yaml" > /dev/null 2>&1
+            if [ $? -eq 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)